src/seedu/addressbook/parser/Parser.java

```diff
@@ -212,7 +212,7 @@ private Command prepareViewAll(String args) {
      * @throws ParseException if no region of the args string could be found for the index
      * @throws NumberFormatException the args string region is not a valid number
      */
-    private int parseArgsAsDisplayedIndex(String args) throws ParseException, NumberFormatException {
+    private static int parseArgsAsDisplayedIndex(String args) throws ParseException, NumberFormatException {
```

Comment:
based on the files changed, I am guessing, you have not edited that part of the code where you create a parser object.
after making this method static, you can remove the parser object from your code!